inter/final-module/Chapter 5/C Miniglk.w

```diff
@@ -112,6 +112,8 @@ void i7_default_glk(i7process_t *proc, i7word_t selector, i7word_t varargc, i7wo
 			rv = i7_miniglk_window_open(proc, a[0], a[1], a[2], a[3], a[4]); break;
 		case i7_glk_set_window:
 			rv = i7_miniglk_set_window(proc, a[0]); break;
+		case i7_glk_window_get_rock:
+			rv = i7_mg_get_window_rock(proc, a[0]); break;
```

Comment:
Some of these functions are called `i7_miniglk`, others `i7_mg`. Is it meant to be public vs private functions? I just left the function name as it was, even though it's now different from the others.